test/tests/functional/pbs_mom_walltime.py

```diff
@@ -83,24 +83,27 @@ def test_hold_time_not_counted_in_walltime(self):
 
         a = {'Resource_List.ncpus': 1}
         J1 = Job(TEST_USER, attrs=a)
-        J1.set_sleep_time(30)
+        J1.set_sleep_time(60)
         jid1 = self.server.submit(J1)
         self.server.expect(JOB, {'job_state': 'R'}, id=jid1)
-        time.sleep(10)
+        # Wait for job to run for sometime
+        time.sleep(15)
         self.server.expect(JOB, {'resources_used.walltime': 0}, op=GT, id=jid1,
                            extend='x')
 
         self.server.holdjob(jid1, USER_HOLD)
         self.server.rerunjob(jid1)
         self.server.expect(JOB, {'Hold_Types': 'u'}, jid1)
-        time.sleep(5)
-
+        # Wait for sometime to verify that this time is not
+        # accounted in 'resource_used.walltime'
+        time.sleep(20)
         self.server.rlsjob(jid1, USER_HOLD)
         self.server.expect(JOB, {ATTR_state: 'F'}, id=jid1, extend='x',
-                           offset=30)
-
-        # The sleep time should not be included in the used walltime
-        self.server.expect(JOB, {'resources_used.walltime': 30}, op=LE,
+                           offset=45)
+        # Verify if the job's walltime is in between 60 to 70
+        self.server.expect(JOB, {'resources_used.walltime': 60}, op=GE,
+                           id=jid1, extend='x')
+        self.server.expect(JOB, {'resources_used.walltime': 70}, op=LE,
```

Comment:
just a thought that can take (job_time + held_time) for LE comparion instead of 70?

Reply:
@dinesh1306 : I have added 10s of grace period considering the min_check_poll value

Reply:
Since you are doing 2 expects, maybe do a status() on the job and do this in an if statement?  It isn't necessary though.  The two expects() work.

Reply:
Thanks @bhroam for the review.

- status() function returns used_walltime in the format of "HH:MM:SS". Inorder to compare the walltime range using if statement, I need to write a function which converts the walltime(returned by status function) into integer and then compare it with if.  I would like to keep it as it is